cmd/scaffold.go

```diff
@@ -1284,12 +1284,12 @@ func (fnb *FlowNodeBuilder) initFvmOptions() {
 		fvm.WithBlocks(blockFinder),
 		fvm.WithAccountStorageLimit(true),
 	}
-	if fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Sandboxnet || fnb.RootChainID == flow.Mainnet {
+	if fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Sandboxnet || fnb.RootChainID == flow.Previewnet || fnb.RootChainID == flow.Mainnet {
```

Comment:
This is getting longer and longer, maybe refactor this to a switch:
```suggestion
	switch fnb.RootChainID {
	case flow.Testnet,
		flow.Sandboxnet,
		flow.Previewnet,
		flow.Mainnet:
```

Reply:
Isn't sandboxnet deprecated now?

Reply:
That's an unrelated to what this PR is about, adding previewnet